response_operations_ui/templates/create-message.html

```diff
@@ -0,0 +1,112 @@
+{% extends "layouts/base.html" %}
+
+{% block page_title %} Create Message | Survey Data Collection {% endblock %}
+
+{% block main %}
+<style>
+    table, th, td {
+    border: 0px;
+    }
+</style>
+
+{% if form.errors %}
+    <div class="panel panel--error">
+        <div class="panel__header">
+            {% if form.errors|length > 1 %}
+                <h1 class="panel__title venus">This page has {{ form.errors|length }} errors</h1>
+            {% elif form.errors %}
+                <h1 class="panel__title venus">This page has {{ form.errors|length }} error</h1>
+            {% endif %}
+        </div>
+        <div class="panel__body">
+            <ol>
+                {% for error in form.errors %}
+                <li class="panel__title venus">{{ form.errors[error][0] }}</li>
+                {% endfor %}
+            </ol>
+        </div>
+    </div>
+    <br />
+{% endif %}
+
+<h1 class="saturn">Create message</h1>
+
+<fieldset>
+
+    <div class="secure-message-form">
+        <form action="{{ url_for('messages_bp.create_message') }}" method="post" id="create-message-form">
+            {{ form.hidden_tag() }}
+
+            <table id="ru-details-table" border="0" cellpadding="0" cellspacing="0" width="40%">
+                <tr>
+                    <td width="10%">Survey:</td><td><b>{{ form.survey(id="survey-label") }}</b></td>
+                </tr>
+                <tr>
+                    <td width="10%">RU Ref:</td><td><b>{{ form.ru_ref(id="ru-ref-label") }}</b></td>
+                </tr>
+                <tr>
+                    <td width="10%">Business:</td><td><b>{{ form.business(id="business-label") }}</b></td>
+                </tr>
+                <tr>
+                    <td width="10%">To:</td><td><b>{{ form.to(id="to-label") }}</b></td>
+                </tr>
+            </table>
+
+
+            {% if form.subject.errors %}
+                <div class="panel panel--simple panel--error panel--spacious" id="subject-error">
+                    <p class="error-message">{{ form.errors['subject'][0] }}</p>
+            {% endif %}
+            <table width="40%">
+                <tr>
+                    <td align="left" valign="bottom">
+                        <label class="venus" for="subject">Subject</label>
+                        <p class="u-vh">Message subject is required</p>
+                    </td>
+                </tr>
+                <tr>
+                    <td align="left" valign="top">
+                        {{ form.subject(id='secure-message-subject',
+                        class_='secure-message-form__subject input input--text',
+                        maxlength='96', size=80) }}
+                    </td>
+                </tr>
+            </table>
+            {% if form.subject.errors %}
+                </div>
```

Comment:
I'm not a fan of this way of doing this.  It seems messy to have 2 {% if error %} blocks.  I might be wrong though and the whole thing needs to be encapsulated in a single div